pynest/nest/lib/hl_api_parallel_computing.py

```diff
@@ -86,3 +86,19 @@ def SetAcceptableLatency(port, latency):
 
     sps(latency)
     sr("/%s exch SetAcceptableLatency" % port)
+
+
+@check_stack
+def SetMaxBuffered(port, size):
+    """Set the maximum buffer size for a MUSIC port.
+
+    Parameters
+    ----------
+    port : object
```

Comment:
So port is a port name, in SLI /port_name? The comment should probably be more specific than "object"